core/modules/cart/components/AddToCart.ts

```diff
@@ -6,6 +6,10 @@ export const AddToCart = {
     product: {
       required: true,
       type: Object
+    },
+    disabled: {
+      required: false,
+      type: Boolean
```

Comment:
if it's not required then we don't have to do `required: false`, but good practice is to provide a default value for that, so:
```js
disabled: {
  type: Boolean,
  default: false
}
```